package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@zenko/cloudserver",
-  "version": "8.4.16",
+  "version": "8.4.15",
```

Comment:
Is it wanted to decrement the version?

Reply:
No, indeed, thank you! I fixed it, and I will fix the conflicts in 8.5 branch right away. In 8.5 the version will be bumped as I will release the project after, for Artesca.

Reply:
Done. Here is the link: https://github.com/scality/cloudserver/compare/w/8.5/bugfix/CLDSRV-286-bump-arsenal-fix-authz-regression